src/Controls/src/SourceGen/LocalVariable.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Maui.Controls.SourceGen;
+
+interface ILocalVariable
```

Comment:
I only see one implementation of this interface (or did I miss it?). What's the plan with `ILocalVariable`?